from .exception_func import division_by_zero